ui/imports/shared/views/NetworkCardsComponent.qml

```diff
@@ -22,24 +22,22 @@ Item {
     property double amountToSend: 0
     property double requiredGasInEth: 0
     property bool errorMode: {
-        if(customMode) {
-            return (d.customAmountToSend > amountToSend) || (d.customAmountToSend < amountToSend) ||
-                    (d.customAmountToReceive > amountToSend) || (d.customAmountToReceive < amountToSend)
-        }
-        else {
-            return  !d.thereIsApossibleRoute
-        }
+        if(amountToSend === 0)
+            return false
+
+        if(!d.thereIsApossibleRoute)
+            return true
+
+        return !(d.customAmountToSend <= root.amountToSend)
```

Comment:
Will this simpler approach work `return d.customAmountToSend > root.amountToSend`?

Reply:
sadly not
cause we can have  d.customAmountToSend > root.amountToSend as false and still there not be any  thereIsApossibleRoute

Reply:
`return !(d.customAmountToSend <= root.amountToSend)` this line alone can be surely inverted... hurts my brain :laughing:

Reply:
lol simplified now